snapcraft/internal/lifecycle.py

```diff
@@ -234,8 +234,9 @@ def _handle_dirty(self, part, step, dirty_report):
                         humanized_options, pluralized_connection))
 
             message_components.append(
-                "\nPlease clean that part's {!r} step in order to "
-                'continue'.format(step))
+                "\nIn order to continue, please clean that part's {0!r} step "
+                "running:\n"
```

Comment:
I think this should be `"by running:\n"` also, it may be better to put it on the same line (no `\n`)